packages/core/src/App/Containers/RealAccountSignup/real-account-signup.jsx

```diff
@@ -277,6 +278,33 @@ const RealAccountSignup = ({
 
     const [assessment_decline, setAssessmentDecline] = React.useState(false);
 
+    const analytic_data = React.useMemo(
+        () => ({
+            form_source: document.referrer,
+            form_name: 'real_account_signup_form',
+            landing_company: real_account_signup_target,
+        }),
+        [real_account_signup_target]
+    );
+
+    const trackEvent = React.useCallback(
+        payload => {
+            if (real_account_signup_target === 'maltainvest') return;
+
+            Analytics.trackEvent('ce_real_account_signup_form', {
+                ...analytic_data,
+                ...payload,
+            });
+        },
+        [analytic_data, real_account_signup_target]
+    );
+
+    React.useEffect(() => {
+        if (is_real_acc_signup_on) {
+            trackEvent({ action: 'open' });
+        }
+    }, [is_real_acc_signup_on, trackEvent]);
```

Comment:
We need action close as well
```suggestion
    React.useEffect(() => {
        if (is_real_acc_signup_on) {
            trackEvent({ action: 'open' });
        }
        return () => trackEvent({ action: 'close' })
    }, [is_real_acc_signup_on, trackEvent]);
```

Reply:
As stated above, the close action is in different file which is triggered before closing the modal for confirmation that the information added will be lost

Reply:
Right, thanks for the explanation @shahzaib-deriv ! 👍